runtime/mangata-kusama/src/weights/extrinsic_weights.rs

```diff
@@ -0,0 +1,76 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-11-09 (Y/M/D)
+//! HOSTNAME: `ns5000592`, CPU: `Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz`
+//!
+//! SHORT-NAME: `extrinsic`, LONG-NAME: `ExtrinsicBase`, RUNTIME: `Mangata Development`
+//! WARMUPS: `10`, REPEAT: `100`
+//! WEIGHT-PATH: ``
+//! WEIGHT-METRIC: `Average`, WEIGHT-MUL: `1.0`, WEIGHT-ADD: `0`
+
+// Executed Command:
+//   ./target/release/mangata-node
+//   benchmark
+//   overhead
+//   --max-ext-per-block
+//   2000
+//   --execution
+//   native
+//   --chain
+//   dev
+
+use frame_support::{
+	parameter_types,
+	weights::{constants::WEIGHT_PER_NANOS, Weight},
+};
+
+parameter_types! {
+	/// Time to execute a NO-OP extrinsic, for example `System::remark`.
```

Comment:
shouldn't this be a full block of NO-OP extrinsics instead a single one ?

Reply:
not sure what that comment is about, it comes from substrate and definately it wasnt benchmarked with single extrinsic but with fully filled block (https://github.com/mangata-finance/substrate/blob/9b401eac46bcccdccd9063e2ebe7c8a966a3078e/utils/frame/benchmarking-cli/src/overhead/weights.hbs#L39)

Reply:
```
target/release/mangata-node benchmark overhead --execution native --chain dev -lblock_builder=debug --base-path .
2022-12-13 08:54:56.237  INFO main frame_benchmarking_cli::extrinsic::bench: creating inherents    
2022-12-13 08:54:56.238  INFO main frame_benchmarking_cli::extrinsic::bench: pushing inherents    
2022-12-13 08:54:56.239  INFO main frame_benchmarking_cli::extrinsic::bench: applying previous block txs    
2022-12-13 08:54:56.239 DEBUG main block_builder: previous block enqueued 0 txs    
2022-12-13 08:54:56.239  INFO main block_builder: executed 0/0 previous block transactions    
2022-12-13 08:54:56.239  INFO main frame_benchmarking_cli::extrinsic::bench: Building block, this takes some time...    
2022-12-13 08:55:02.322 DEBUG main block_builder: consume 1332 valid transactios    
2022-12-13 08:55:02.323  INFO main frame_benchmarking_cli::extrinsic::bench: Extrinsics per block: 1481    
2022-12-13 08:55:02.323  INFO main frame_benchmarking_cli::extrinsic::bench: Running 10 warmups...    
2022-12-13 08:55:05.107  INFO main frame_benchmarking_cli::extrinsic::bench: Executing block 100 times
```